command/agent/agent.go

```diff
@@ -256,19 +258,19 @@ func (c *cmd) run(args []string) int {
 	}
 
 	if err := c.startupJoin(agent, config); err != nil {
-		c.logger.Error((err.Error()))
+		c.logger.Error(err.Error())
 		return 1
 	}
 
 	if err := c.startupJoinWan(agent, config); err != nil {
-		c.logger.Error((err.Error()))
+		c.logger.Error(err.Error())
 		return 1
 	}
 
 	// Let the agent know we've finished registration
 	agent.StartSync()
 
-	cli.output("Consul agent running!")
+	c.logger.Info("Consul agent running!")
```

Comment:
This also changed to use the logger (as mentioned in the commit message). Previously this was leading to very odd looking output, where the log lines were broken up by a random `==> Consul agent running!"` line.

With this change the message will be included in the logs when json logs are enabled.